public/locales/fr-FR.json

```diff
@@ -6,7 +6,7 @@
   "Docs": "Documentation",
   "Voting": "Votes",
   "Lottery": "Loterie",
-  "Connect Wallet": "Connecter un portefeuille",
+  "Connect Wallet": "Se connecter",
```

Comment:
Aha is this a bit of a DIY-translation fix? I think you may need to make the change through Crowdin for it to stick. @hachiojidev would be able to confirm